go.mod

```diff
@@ -0,0 +1,168 @@
+	github.com/vincent-petithory/dataurl v0.0.0-20191104211930-d1553a71de50
+	github.com/vmware/govmomi v0.22.1
+	go.uber.org/atomic v1.5.1 // indirect
+	go.uber.org/multierr v1.4.0 // indirect
+	go.uber.org/zap v1.13.0 // indirect
+	golang.org/x/crypto v0.0.0-20191206172530-e9b2fee46413
+	golang.org/x/lint v0.0.0-20191125180803-fdd1cda4f05f
+	golang.org/x/oauth2 v0.0.0-20191202225959-858c2ad4c8b6
+	golang.org/x/sys v0.0.0-20191206220618-eeba5f6aabab
+	golang.org/x/time v0.0.0-20191024005414-555d28b269f0 // indirect
+	golang.org/x/xerrors v0.0.0-20191204190536-9bdfabe68543 // indirect
+	google.golang.org/api v0.14.0
+	gopkg.in/AlecAivazis/survey.v1 v1.8.7
+	gopkg.in/ini.v1 v1.51.0
+	gopkg.in/yaml.v2 v2.2.7
+	k8s.io/api v0.17.1
+	k8s.io/apimachinery v0.17.1
+	k8s.io/client-go v12.0.0+incompatible
+	k8s.io/klog v1.0.0
+	k8s.io/kube-openapi v0.0.0-20191107075043-30be4d16710a // indirect
+	k8s.io/utils v0.0.0-20191114200735-6ca3b61696b6
+	sigs.k8s.io/cluster-api-provider-aws v0.4.7
+	sigs.k8s.io/cluster-api-provider-azure v0.2.2
+	sigs.k8s.io/cluster-api-provider-openstack v0.2.0
+)
+
+replace (
+	github.com/Sirupsen/logrus => github.com/sirupsen/logrus v1.3.0 // Pulled in by tf-vsphere-provider
+	github.com/coreos/go-systemd => github.com/coreos/go-systemd v0.0.0-20181031085051-9002847aa142 // Pin non-versioned import v22.0.0
+	github.com/go-log/log => github.com/go-log/log v0.1.1-0.20181211034820-a514cf01a3eb // Pinned by MCO
+	github.com/hashicorp/consul => github.com/hashicorp/consul v1.6.2 // Pin to version required by terraform
+	github.com/hashicorp/terraform => github.com/LorbusChris/terraform v0.12.18-0.20200119232315-d11f8e438757 // Pin to fork with deduplicated rpc types
+	github.com/hashicorp/terraform-plugin-sdk => github.com/LorbusChris/terraform-plugin-sdk v1.4.1-0.20191218013551-33541515cfe4 // Pin to fork with deduplicated rpc types
+	github.com/kubernetes-sigs/aws-iam-authenticator => github.com/kubernetes-sigs/aws-iam-authenticator v0.3.1-0.20181019024009-82544ec86140 // Pin to version required by tf-provider-aws
+	github.com/metal3-io/baremetal-operator => github.com/openshift/baremetal-operator v0.0.0-20191209182400-81ee99999340 // Use OpenShift fork
+	github.com/metal3-io/cluster-api-provider-baremetal => github.com/openshift/cluster-api-provider-baremetal v0.0.0-20190821174549-a2a477909c1d // Pin OpenShift fork
+	github.com/openshift/api => github.com/openshift/api v0.0.0-20200117162508-e7ccdda6ba67 // Pin API
+	github.com/openshift/machine-config-operator => github.com/openshift/machine-config-operator v0.0.1-0.20200102225717-04cd2198cae2 // Pin MCO so it doesn't get downgraded
+	github.com/terraform-providers/terraform-provider-vsphere => github.com/LorbusChris/terraform-provider-vsphere v1.14.1-0.20200128145507-331c3c135db7 // Pin to fork that has migrated to tf-plugin-sdk
+	google.golang.org/api => google.golang.org/api v0.11.1-0.20191012001332-5e0e3f4a3bb8 // Pin to version required by tf-provider-google
```

Comment:
can we drop our require to match the tf-provider-google instead of replace ?

Reply:
This slipped - I should've updated the pin to 0.13.0 with the google provider bump. However we do still need the replace directive here, as it would be automatically bumped to 0.14.0 otherwise.